MAGEDOC-3473: Add 2.0 build steps to the README (#4648)
magento/devdocs: README.md

> Â# Magento Developer Documentation

Reviewer: Odd little A seems to made it in this line?

Author: How'd that get there?! Fixed it.

> If you don't have the `_config.local.yml` file at the root of your `devdocs/` directory, the rake will generate all versions of the documentation.

Reviewer 2: ```suggestion
If you do not have the `_config.local.yml` file at the root of your `devdocs/` directory, the rake will generate all versions of the documentation.
```

Author: Out of scope

> ### To build v2.0 documentation

Reviewer: Other sections start with the action verb Build, maybe keep this Build v2.0 documentation?

Author: No, I'd rather keep this heading consistent with the heading the immediately precedes this section.

Magento 2.0.18 was the final 2.0.x release. After March 2018, Magento 2.0.x stopped receiving security patches, quality fixes, and documentation updates.

To ensure the quality of our public documentation for versions of Magento that are still supported and to avoid potential confusion about 2.0.x support, we removed all 2.0.x content from our public documentation websites on May 31, 2019.

To build v2.0 documentation:

> > You can also use the GitHub interface to read the source Markdown files instead of building the site locally. Just navigate to the [`2.0`](https://github.com/magento/devdocs/tree/2.0) branch and click the file you want to view.

Reviewer: Slight edit option for easier translating:

```suggestion
> You can also use the GitHub interface to read the source Markdown files instead of building the site locally. Navigate to the [`2.0`](https://github.com/magento/devdocs/tree/2.0) branch and click the file you want to view.
```

1. Add an entry for 2.0 in the `Docfile.yml` file.

    ```yaml
    -
      directory: guides/v2.0
      repository: magento/devdocs
      branch: 2.0
      filter: false
    ```

1. Initialize the `2.0` branch in your local devdocs repository.

    ```shell
    rake init
    ```

> 1. Run the preview command.

Reviewer 2: You could say what the command does...
```suggestion
1.  Build a preview of the documentation site.
```

Author: Rejected. I prefer to be consistent with the section immediately preceding this section. We can open a separate PR to clean up the entire file.

Reviewer 2: We've tried that before.

    ```shell
    rake preview
    ```

> 

Reviewer: Maybe add a note that when complete, it will open the documentation in a web browser? Sometimes the build can take time, giving them some expectation of completion could be helpful for first time builders.
